tracker/data/processing.py

```diff
@@ -579,16 +619,17 @@ def https_behavior_for(name, pshtt, sslyze, parent_preloaded=None):
         # ITPIN cares about usage of TLS 1.0 and TLS 1.1
         tlsv10 = boolean_for(sslyze["TLSv1.0"])
         tlsv11 = boolean_for(sslyze["TLSv1.1"])
+        uses_accepted_ciphers = all(cipher in ACCEPTED_CIPHERS for cipher in sslyze.get("Accepted Ciphers").split(', '))
```

Comment:
Are you going to pull out the ciphers that the service is using that we don't support? or did that get fixed up somewhere else?